### PR TITLE
feat(api): add support for bulk imports API

### LINKED ISSUE #2414: Add support for bulk imports API
See https://docs.gitlab.com/ee/api/bulk_imports.html

### PR DESCRIPTION
**Pull request by nejch**
Closes #2414

### CODE REVIEW

File: tests/functional/api/test_bulk_imports.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    assert created_migration.status == "created"
+
+    migration = gl.bulk_imports.get(created_migration.id)
+    assert migration == created_migration
+
+    migrations = gl.bulk_imports.list()
+    assert migration in migrations
+
+    migration.refresh()
+    assert migration.status == "started"
```
Comment (nejch) [inline tests/functional/api/test_bulk_imports.py:30]: This will probably be flaky in CI. I'll change it.

Comment (nejch) [reply tests/functional/api/test_bulk_imports.py]: Done with 841a517a1f03c0a8ececf7f0f9a54d2ff314dccc, also added unit tests.

**Comment by nejch** [comment]
@lmilbaum if you like and have some spare time, you could also take a look at this one. If I have an approval I'll merge before the next release ;)

**Review by lmilbaum — APPROVED** [review]
LGTM

**Comment by nejch** [comment]
Thanks a lot for the review @lmilbaum!

**Pull request merged**